Cargo.toml

```diff
@@ -11,16 +11,18 @@ cosmic-randr = { git = "https://github.com/pop-os/cosmic-randr" }
 tokio = { version = "1.40.0", features = ["macros"] }
 
 [workspace.dependencies.libcosmic]
-git = "https://github.com/pop-os/libcosmic"
 features = [
```

Comment:
If some of these features can be made optional but default, then it is likely cosmic-settings would run fine on any OS.

Reply:
ok, i can probably do that now.

Reply:
I added these as default, but they can be disabled "wayland", "dbus-config", "single-instance", "a11y", "wgpu"